app/src/main/res/values/dimens.xml

```diff
@@ -6,6 +6,7 @@
     <dimen name="default_margin">16dp</dimen>
     <!-- Half margin of default_margin. Can be helpful if we items next to each other. Eg: RecyclerView. -->
     <dimen name="default_margin_half">8dp</dimen>
+    <dimen name="text_margin">16dp</dimen>
```

Comment:
I haven't seen where you use that. But use `default_margin` instead and remove please

Reply:
deleted line 9